src/features/dashboard/components/PortfolioHistoryChart.tsx

```diff
@@ -0,0 +1,149 @@
+import { CartesianGrid, Line, LineChart, XAxis, YAxis } from 'recharts';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import {
+  ChartConfig,
+  ChartContainer,
+  ChartTooltip,
+  ChartTooltipContent,
+} from '@/components/ui/chart';
+import { useState } from 'react';
+import { Button } from '@/components/ui/button';
+
+//to be replaced with real data
+const chartData = [
```

Comment:
Mocked data should live outside of a component in a separate ts file. In this case this data spreads over a significant number of rows so it would be beneficial. It it's just a few lines it's kid of OK.